source/extensions/filters/common/expr/context.h

```diff
@@ -194,13 +196,15 @@ class ResponseWrapper : public BaseWrapper {
   ResponseWrapper(Protobuf::Arena& arena, const ::Envoy::Http::ResponseHeaderMap* headers,
                   const ::Envoy::Http::ResponseTrailerMap* trailers,
                   const StreamInfo::StreamInfo& info)
-      : BaseWrapper(arena), headers_(arena, headers), trailers_(arena, trailers), info_(info) {}
+      : BaseWrapper(arena), headers_(arena, headers), trailers_(arena, trailers), info_(info),
+        timing_(info) {}
   absl::optional<CelValue> operator[](CelValue key) const override;
 
 private:
   const HeadersWrapper<::Envoy::Http::ResponseHeaderMap> headers_;
   const HeadersWrapper<::Envoy::Http::ResponseTrailerMap> trailers_;
   const StreamInfo::StreamInfo& info_;
+  Envoy::StreamInfo::TimingUtility timing_;
```

Comment:
I think this is a light utility object, you can stack allocate it during computation instead of storing it.

Reply:
DONE.